lib/cleanup_group.c

```diff
@@ -62,7 +62,7 @@ void cleanup_report_mod_group (void *cleanup_info)
 	         gr_dbname (),
 	         info->action));
 #ifdef WITH_AUDIT
-	audit_logger (AUDIT_USER_ACCT, log_get_progname(),
+	audit_logger (AUDIT_GRP_MGMT, log_get_progname(),
```

Comment:
Could you please explain the change `AUDIT_USER_ACCT` => `AUDIT_GRP_MGMT` in the commit message?

Reply:
Done